dbcp2-and-tomcat/run.sh

```diff
@@ -0,0 +1,24 @@
+#/bin/bash
+set -e
+
+export QUICKSTART_NAME=${PWD##*/}
+TOMCAT_VERSION=7.0.82
+wget -nc https://archive.apache.org/dist/tomcat/tomcat-7/v$TOMCAT_VERSION/bin/apache-tomcat-$TOMCAT_VERSION.zip
+rm -rf apache-tomcat-$TOMCAT_VERSION
+unzip apache-tomcat-$TOMCAT_VERSION.zip
+export TOMCAT_HOME=$(pwd)/apache-tomcat-$TOMCAT_VERSION/
+chmod +x $TOMCAT_HOME/bin/catalina.sh
+mvn package
+rm -rf $TOMCAT_HOME/webapps/${QUICKSTART_NAME}/
+cp target/${QUICKSTART_NAME}.war $TOMCAT_HOME/webapps/
+JPDA_SUSPEND=n $TOMCAT_HOME/bin/catalina.sh jpda run &
+sleep 10
+
+for i in {1..10}
+do
+  curl -f --data "test$i" http://localhost:8080/${QUICKSTART_NAME}
+done
```

Comment:
Would it be possible to add in something that calls the crash endpoint and then the recover one and verifies the json has the new string dao please?

Reply:
yeah, I will do